### DISCUSSION TITLE
System crashes when updating.

### DISCUSSION THREAD

**Discussion by manu20247** [helloSystem/hello#566] [General]
Good morning!  I am having difficulty updating the system.  After the update is complete and I restart the system, it simply crashes with the broken Xorg.  I go into command line mode and type start X and I get the response that there was an error starting the graphical server.  This only happens after updating the entire system.  I find it strange because a long time ago this didn't happen.  Updating freeBSD is breaking the system.  And another thing is happening, after updating many packages a note appears in the command terminal saying that there are no longer any Port maintainers of that updated package.  No longer supported or updated.

And hello OS 0.7 when updating no longer finds the Hello Desktop repositories 😞

Thank you and have a great day everyone.

**Comment by probonopd** [comment]
Yes. The whole package management based approach is too flaky for my taste. We should ideally find a way to make an "immutable system" (image based, where end users do not update individual packages but the whole OS image at once = atomic).
